.travis.yml

```diff
@@ -169,11 +169,14 @@ matrix:
 before_script:
   # Bootstrap everything (including letor, which isn't done
   # by default), then configure using our chosen compiler.
+  - eval "$(gimme 1.14)"
+  - eval env
   - travis_retry ./bootstrap xapian-core xapian-applications/omega swig xapian-bindings xapian-letor
   - ./configure $confargs CC="$USE_CC" CXX="$USE_CXX"
 script:
   - make -j2
   - $installcore
+  #  - sudo env "PATH=$PATH" make install
```

Comment:
This commented out code looks like a left-over from debugging/development.

Reply:
yea ill remove it